valuable/src/enumerable.rs

```diff
@@ -130,6 +130,8 @@ impl fmt::Debug for dyn Enumerable + '_ {
                         self.fmt.field(field.name(), value);
                     }
                 }
+
+                fn visit_value(&mut self, _: Value<'_>) {}
```

Comment:
should these also be `unreachable!()` (or maybe just panic only in debug mode)? my concern here is just that if we accidentally break something that the value is wrongly dispatched to `visit_value`, we would essentially swallow the value and do nothing...

this is probably not actually important though.

Reply:
done